index.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
         ],
       },
     ],
+    "no-only-tests/no-only-tests": "error",
```

Comment:
Don't we need to also add the `no-only-tests` to `plugins`? That's what I'm getting from the Usage section in [the npm package description](https://www.npmjs.com/package/eslint-plugin-no-only-tests).

Reply:
If I remember correctly, when I was testing it worked without adding the dependency. Would be awesome if you could confirm it locally.

Reply:
I'm getting different behavior when `no-only-tests` is listed in `plugins` vs when it's not.
I tested this way:
1. I've changed `it(` to `it.only(` in one of tests in `tbtc-v2/solidity/test/bridge/Bridge.MovingFunds.test.ts`.
2. I've set the `@thesis-co/eslint-config` dependency in `tbtc-v2/solidity/package.json` to `github:thesis/eslint-config#no-only-tests`.
3. I've removed the `tbtc-v2/solidity/node_modules/@thesis-co/eslint-config` and run `yarn upgrade @thesis-co/eslint-config`.
4. I've run `yarn lint` in `tbtc-v2/solidity`

As a result, I've got `✖ 141 problems (127 errors, 14 warnings)`, most of them were `Definition for rule 'no-only-tests/no-only-tests' was not found`.

When I did the same, but set up the dependency in step 2 to my local `eslint-config` which was based on `no-only-tests` branch plus had the plugin added to the `plugins`, the result of linting was following: `✖ 15 problems (1 error, 14 warnings)` and that one error was `it.only not permitted`.

Based on that I think we need to include `no-only-tests` in `plugins` in `index.js`.

Reply:
Yeah, looks like you are correct. I updated the code.